solidity/random-beacon/tasks/index.ts

```diff
@@ -1,4 +1,4 @@
 // eslint-disable-next-line global-require
 require("./unlock-eth-accounts")
```

Comment:
Let's revert changes to this file as it fails linting anyway.

Reply:
[daf9e3e](https://github.com/keep-network/keep-core/pull/3096/commits/daf9e3eb620e017157df9ef9db0bef35da7c4fb5)